include/rxmesh/matrix/dense_matrix.cuh

```diff
@@ -10,15 +10,25 @@ namespace rxmesh {
 // Currently this is device only
 // host/device transormation will be added
 // only col major is supported
```

Comment:
I guess this should change since we now support dense matrix on both host and device